owncloud/owncloud.py

```diff
@@ -234,6 +246,10 @@ def list(self, path):
             return res[1:]
         return None
 
+    def webdav(self, cmd, path):
```

Comment:
Not sure why this is needed ? Hopefully most APIs are covered by other calls already.